src/cli/commands/start.ts

```diff
@@ -6,19 +6,18 @@ import shell from "shelljs";
 import builder from "../../builder";
 import { Dashboard } from "../../dashboard";
 import { createRuntimeHost } from "../../runtimeHost";
-import { GithubActionSWAConfig, isPortAvailable, parseUrl, readConfigFile } from "../../utils";
+import { GithubActionSWAConfig, isAcceptingTcpConnections, isHttpUrl, isPortAvailable, parseUrl, readConfigFile } from "../../utils";
 import { DEFAULT_CONFIG } from "../config";
 
 export async function start(startContext: string, program: CommanderStatic) {
-  if (startContext.startsWith("http")) {
+  if (isHttpUrl(startContext)) {
```

Comment:
Just in case it's a folder that starts with `http`.

Reply:
Good catch!